### DISCUSSION TITLE
夸奖一下

### DISCUSSION THREAD

**Discussion by cqyisbug** [imhuzi/qiye-wechat#6] [General]
相比于wxjava，这个项目的实现太干净了。

**Comment by cqyisbug** [comment]
虽然项目只是完成了一半，哈哈哈，有兴趣一起做一下吗？你的技术栈和我的好像。

**Reply by imhuzi** [reply]
哈哈，有人回应有动力了，这周争取把所有api 完善，发一版仓库吧，有什么需求可以 提出来大家一起完善

**Reply by cqyisbug** [reply]
> 哈哈，有人回应有动力了，这周争取把所有api 完善，发一版仓库吧，有什么需求可以 提出来大家一起完善

关注好久了呀，目前在想获取各个token的实现，感觉要继承下JacksonDecoder，抛一个Retry异常。

**Reply by cqyisbug** [reply]
token那块实现好了，再把事件机制完善应该就差不多了。
